src/collection-view.js

```diff
@@ -644,18 +693,15 @@ const CollectionView = Backbone.View.extend({
   },
 
   // Destroy the child views that this collection view is holding on to, if any
-  _destroyChildren({checkEmpty} = {}) {
-    this.triggerMethod('before:destroy:children', this);
-    const shouldCheckEmpty = checkEmpty !== false;
+  _destroyChildren(options) {
     const childViews = this.children.map(_.identity);
 
-    this.children.each(_.bind(this.removeChildView, this));
-
-    if (shouldCheckEmpty) {
```

Comment:
I reverted all the changes to this function, except the use `_removeChildViews`,  and several test regarding `_checkEmpty` broke. I think I will have to leave the changes I made.